File: Data Structures & Algorithms/majority-element/submission-2.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        best_num =None
        best_count = 0
        for num in nums:
           if nums.count(num)> best_count:
                best_count = nums.count(num)
                best_num = num
        return  best_num

        

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.ts
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    longestConsecutive(nums: number[]): number {
        const numbers = new Set(nums)
        let startingNum = 0
        let longest = 0
        for(let i=0;i<=nums.length;i++){
            let num = nums[i]
            if(!numbers.has(num-1)){
                startingNum = num
                let counter = 0
                let index = 0
                while(numbers.has(startingNum + index)){
                    index++
                    counter++
                    longest = Math.max(longest,counter)
                }
            }
        }
        return longest
    }
}
